src/client/Microsoft.Identity.Client/json/.editorconfig

```diff
@@ -0,0 +1,7 @@
+﻿# Code files
```

Comment:
Good idea to add .editorconfig here.

Reply:
It doesn't really matter, as long as the build is green and 2 reviewers have OK-ed the change, it's all good. I started using the auto-merge option more.

![image](https://user-images.githubusercontent.com/12273384/180434125-15594449-0457-4e16-927c-651450c9e6eb.png)